caffe2/utils/threadpool/pthreadpool-cpp.cc

```diff
@@ -59,6 +93,12 @@ size_t getDefaultNumThreads();
 PThreadPool* pthreadpool() {
   static std::unique_ptr<PThreadPool> threadpool =
       std::make_unique<PThreadPool>(getDefaultNumThreads());
+  #ifndef WIN32
+    // Register pthread_atfork handlers to prevent segfaults in worker processes
+    // after fork.
+    static std::once_flag flag;
+    std::call_once(flag, []{ pthread_atfork(before_fork, after_fork, NULL); });
```

Comment:
At 3 places in the codebase, `pthread_atfork` is being used (for the child process handler, which won't conflict with this one), and 2 of them use `pthread_atfork` in `std::call_once`, but I'm not sure if it's needed here as well.